tests/test_biolink_model/test_biolink_model.py

```diff
@@ -57,7 +57,11 @@
 )
 def test_biolink(generator, extension, gen_kwargs, serialize_kwargs, temp_dir, snapshot, input_path):
     BIOLINK_YAML = input_path("biolink-model.yaml")
-    if not extension.startswith("."):
+    if extension == ".graphql":
+        with pytest.raises(NotImplementedError):
+            generated = generator(BIOLINK_YAML, **gen_kwargs).serialize(**serialize_kwargs)
```

Comment:
so this will look like a passing test, rather than a reminder that it's something we should fix, will suggest xfailing

Reply:
ope i guess i can't make suggestions on lines that weren't touched in the PR. 

would recommend adding `marks=pytest.mark.xfail` to the `GraphqlGenerator` param on L49

Reply:
I have misunderstood something. I thought it should not fail, but rather raise a warning and generate without the empty enum.

And that is was we have back. The resulting GraphQL is invalid if that enum is used somewhere else, but #2343 should address it, right?

Reply:
Ah ya, if it doesn't fail (but emits nothing?) Then yes good! These comments were just saying "if we were going to leave this raising an exception so we leave ourselves a reminded that we should fix that later instead of passing the test which is an indication all is well"